tests/test_cli.py

```diff
@@ -163,8 +163,7 @@ def test_install_with_gitlab_app(self):
             tests_root,
             "cached_nulecules/gitlab/.workdir")
 
-        assert set(os.listdir(work_dir)) == \
-            set(["gitlab", "postgresql", "redis"])
+        assert os.path.isdir(work_dir) == False
```

Comment:
This `assert` adds the ordering constraint. If this test is moved after `run` it will fail -because the `run` will create the `.workdir`